notebooks/lesson04_intro-to-random-forests.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>do you have the solutions to exercises here on purpose?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>will be deleted in next push</p>